InstallWindows.md

```diff
@@ -149,6 +149,22 @@ You'll be able to recognize the partitions we made earlier by their size. take n
 
 **_WARNING: We'll assume X: is the Win partition and that Y: is the ESP partition for the next commands. Replace them correctly or you'll lose data on your PC._**
 
+- Presuming you are working on a Windows 64-bit machine, download and extract win64.zip file from the UUPMediaCreator repo (listed above). Proceed to directory: `"win-x64\CLI" `
+
+-  Inside the CLI directory, open command prompt and run the following command:
+```
+uupdownload -s Professional -v 10.0.19043.1 -r Retail -b Retail -c vb_release -t arm64 -l en-US
```

Comment:
Just a nit: you might want to add -e Professional so it only downloads files needed for Pro, ends up being faster to download/less space wasted.